unitary/examples/quantum_chinese_chess/enums.py

```diff
@@ -42,31 +47,38 @@ class MoveVariant(enum.Enum):
     CAPTURE = 3
 
 
-class Piece(enum.Enum):
-    EMPTY = "."
-    SOLDIER = "s"
-    CANNON = "c"
-    ROOK = "r"
-    HORSE = "h"
-    ELEPHANT = "e"
-    ADVISOR = "a"
-    GENERAL = "g"
+class Color(enum.Enum):
+    NA = 0
+    RED = 1
+    BLACK = 2
 
-    @classmethod
-    def type_of(cls, c: str) -> Optional["Piece"]:
-        return {
-            "s": Piece.SOLDIER,
-            "c": Piece.CANNON,
-            "r": Piece.ROOK,
-            "h": Piece.HORSE,
-            "e": Piece.ELEPHANT,
-            "a": Piece.ADVISOR,
-            "g": Piece.GENERAL,
-            ".": Piece.EMPTY,
-        }.get(c.lower(), None)
 
-    def red_symbol(self) -> str:
-        return self.value.upper()
+class Type(enum.Enum):
+    """The four values are symbols corresponding to
+    English red
+    English black
+    Chinese red
+    Chinese black
+    """
 
-    def black_symbol(self) -> str:
-        return self.value
+    EMPTY = [".", ".", ".", "."]
+    SOLDIER = ["s", "S", "兵", "卒"]
+    CANNON = ["c", "C", "炮", "砲"]
+    ROOK = ["r", "R", "车", "車"]
+    HORSE = ["h", "H", "马", "馬"]
+    ELEPHANT = ["e", "E", "象", "相"]
+    ADVISOR = ["a", "A", "士", "仕"]
+    GENERAL = ["g", "G", "将", "帥"]
+
+    @staticmethod
+    def type_of(c: str) -> Optional["Type"]:
```

Comment:
This function is no longer correct, it won't handle the Chinese symbols, correct?

Reply:
Thanks Doug. This method (currently) is only used for convert FEN symbols into board pieces. I will rewrite it if further needs merge.